client/cityrepdata.cpp

```diff
@@ -726,63 +726,83 @@ static QString cr_entry_cma(const struct city *pcity, const void *data)
 #define FUNC_TAG(var) cr_entry_##var, #var
 
 static const struct city_report_spec base_city_report_specs[] = {
-    {true, -15, 0, nullptr, N_("?city:Name"), N_("City Name"), nullptr,
+    // CITY columns
+    {true, -15, 0, nullptr, N_("?city:Name"), N_("City: Name"), nullptr,
      FUNC_TAG(cityname)},
-    {false, -15, 0, nullptr, N_("Nation"), N_("Nation"), nullptr,
+    {false, -15, 0, nullptr, N_("Nation"), N_("City: Nation"), nullptr,
      FUNC_TAG(nation)},
-    {true, 2, 1, nullptr, N_("?size [short]:Sz"), N_("Size"), nullptr,
+    {false, 3, 1, nullptr, N_("?Continent:C"), N_("City: Continent number"),
```

Comment:
There is another column with C as the label when shown - content citizens. Wonder if this one should be changed to CN or something.